freyja_controller.launch

```diff
@@ -29,8 +29,7 @@
 
   <!-- autostart rosbag -->
 	<arg name="start_rosbag" default="false"/>
-	
-	
+
```

Comment:
Unused arguments: `waypoint_handler_mode`, `example_trajectory`. Argument `example_number` should be announced here as well ..

Reply:
Sorry, this must have been removed in the merge. Fixed

Reply:
I think during the merge `waypoint_handler_mode`, `example_trajectory` were added and `example_number` removed. Sorry about that.